File: 1944-truncate-sentence/truncate-sentence.py
class Solution:
    def truncateSentence(self, s: str, k: int) -> str:
        index=0
        space=0
        for c in s:
            if(c==' '): space=space+1
            if(space==k): break
            index=index+1
        return s[0:index]
        

File: 1944-truncate-sentence/truncate-sentence.cpp
class Solution {
public:
    string cal (string s,int k)
    {
        int index=0,space=0;
        for(char c : s)
        {
            if(c==' ') space++;
            if(space == k) break;
            index++;
        }
        return s.substr(0,index);
    }
    string truncateSentence(string s, int k) 
    {
      return cal(s,k);
    }
};